src/istgt_iscsi.c

```diff
@@ -6019,7 +6027,7 @@ worker(void *arg)
 	conn->exec_lu_task = NULL;
 	lu_task = NULL;
 
-	pthread_cleanup_push(worker_cleanup, conn);
+//	pthread_cleanup_push(worker_cleanup, conn);
```

Comment:
do we still need this change? we should call worker_cleanup.

Reply:
we don't need it. Since this is being popped out on successful exit of worker thread, and we don't return from between.